Dependencies/xr/Source/ARKit/XR.mm

```diff
@@ -1295,10 +1550,10 @@ void GetHitTestResultsLegacy(std::vector<HitResult>& filteredResults, xr::HitTes
         return false;
     }
 
-    bool System::Session::TrySetMeshDetectorEnabled(const bool)
+    bool System::Session::TrySetMeshDetectorEnabled(const bool enabled)
     {
-        // TODO
-        return false;
+        m_impl->TrySetMeshDetectorEnabled(enabled);
+        return enabled;
```

Comment:
>enabled [](http://example.com/codeflow?start=15&length=7)

this should probably always return true compared to return the value provided. returning true when setting the mesh detector to disabled will probably be a utilized scenario in the future. Right now it would return false.

Reply:
actually looks like all the underlying TrySetMeshDetectorEnabled functions also return bools. We should consider just returning the function call here compared to an additional value.